### DISCUSSION TITLE
WebView2_Child process failed ExitCode:-1073740760

### DISCUSSION THREAD

**Discussion by ZhouDev-cloud** [MicrosoftEdge/WebView2Feedback#3984] [Q&A]
After updating WebView2 to 104.0.1293.63, Cisco WebEx cannot be started.（Stuck on the WebEx login screen, spinning all the time.）
There was no problem when using WebView2 ver.100.0.1185.44 before.
What information is needed to solve this problem?

Child process failed
Kind:COREWEBVIEW2_PROCESS_FAILED_KIND_RENDER_PROCESS_EXITED
Reason:COREWEBVIEW2_PROCESS_FAILED_REASON_UNEXPECTED
Exit code:-1073740760
Process description:

Version
Runtime: 104.0.1293.63
OS: Win10

**Comment by LiangTheDev** [comment]
-1073740760 is STATUS_INVALID_IMAGE_HASH, suggesting that there are third party software injecting dlls into WebView renderer process.

You can use Event Viewer (eventvwr.exe) to check Error events under Microsoft-Windows-CodeIntegrity/Operational. Use the Details tab so that we can see whether an event is for msedgewebview2.exe.
As it is likely that the third party software injects into all processes, you can also attach a debugger to one of the msedgewebview2.exe process and then see which non Microsoft modules (dlls) are loaded into the process.

Once we identified the dll, try to find which third party software it is coming from and whether there is any settings in that software to exclude msedgewebview2.exe from injecting target.

If all not working, you can set environment variables WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS with value of --disable-features=RendererCodeIntegrity to disable the Edge security feature so that WebView2 can run. It is not recommended, but that would get it working.

**Comment by ZhouDev-cloud** [comment]
Hi,Liang.Thank you for your answer.
I will try as you explained.
If there is no improvement, I will contact you again.
